docs/releases/adding-new-firmwares.md

```diff
@@ -2,19 +2,17 @@
 
 In case we are about to release both Suite and firmwares we want to add the signed binaries during the Freeze so QA has the whole thing to test.
 
-Binaries and release definitions are stored in `packages/connect-common/files/firmware/*` in the same data structure as they were in the past (data.trezor.io). They are used by `suite-web` and `suite-desktop` builds.
+Binaries and release definitions are stored in `packages/connect-common/files/firmware/*` in the same folder structure as they are in [https://github.com/trezor/webwallet-data](https://github.com/trezor/webwallet-data) deployed on [https://data.trezor.io](https://data.trezor.io). Latest and intermediary firmware binaries are bundled in `suite-desktop` so the whole onboarding process including FW installation can be completed offline. As opposed to the `suite-web` which is downloading FW binaries from [https://data.trezor.io](https://data.trezor.io).
```

Comment:
> As opposed to the `suite-web` which is downloading FW binaries from [https://data.trezor.io](https://data.trezor.io).

This part is probably not correct. I just tried that, and binary was fetch from `https://suite.corp.sldev.cz/suite-web/chore/5440-document-releases-json/web/static/connect/data/firmware/1/trezor-1.11.1.bin`

Reply:
This describes the current situation better: https://github.com/trezor/trezor-suite/pull/5595/commits/786a12a4c1d90fa7b2d456390882ea3b35b82a06